ajax/var_type.js

```diff
@@ -0,0 +1,23 @@
+const log = console.log,
```

Comment:
no need to assign javascript inbuild method to variable

Reply:
Please tell alternative solution